pkg/sfu/rtpmunger.go

```diff
@@ -88,6 +88,7 @@ func (r *RTPMunger) PacketDropped(extPkt *buffer.ExtPacket) {
 
 	r.highestIncomingSN = extPkt.Packet.SequenceNumber
 	r.snOffset += 1
+	r.lastSN = extPkt.Packet.SequenceNumber - r.snOffset
```

Comment:
This little line caused me infinite head ache. When we were dropping packets in temporal layer, this caused a hole/overlap and client went berserk declaring a bunch of loss and sending PLIs. Stark reminder: all it takes is off by a single bit at times and things could get messy.